src/html/js/user.js

```diff
@@ -765,8 +765,8 @@ function closeHelpConfirm(f) {
 }
 
 function viewInfoDisclosureDetail(type){
-    // dummy function
-    console.log(type);
+    $("#viewInfoDisclosureDetail #pageTitle").html(i18next.t("profile." + type));
```

Comment:
If you want the element to be translated dynamically, it is better to do the following.  

         $('#viewInfoDisclosureDetail #pageTitle')
              .attr('data-i18n', 'profile.'+ type)
              .localize();